core-feature-pack/common/src/main/resources/content/bin/standalone.sh

```diff
@@ -144,7 +144,7 @@ if $linux; then
     for var in $CONSOLIDATED_OPTS
     do
        # Remove quotes
-       p=`echo $var | tr -d "'"`
+       p=`echo $var | tr -d "'"` | tr -d "\""
```

Comment:
Ups, this is wrong, it was fixed in the domain.sh version but I forgot to correct it on the standalone one.

The last `tr` must be inside the backticks